### PR TITLE
[15.0][FIX] web_responsive: fix keyboard navigation in Apps menu Search results

### PR DESCRIPTION
**Pull request by SplashS**
Due to changes in javascript framework keyboard navigation in Search results was broken.
This PR makes it work again.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Any step to reproduce the issue before the patch for testing this?

**Comment by SplashS** [comment]
> Any step to reproduce the issue before the patch for testing this?

Open AppsMenu
Search something
Press ArrowDown on keyboard to navigate results
This doesn’t work

**Comment by pedrobaeza** [comment]
@sbidoul Last week I'm not able to try anything in runboat. It gives an nginx 503 error. Do you know why?

File: web_responsive/static/src/components/apps_menu/apps_menu.xml [hunk 1]
```diff
@@ -6,54 +6,67 @@
     <t t-inherit="web.NavBar.AppsMenu" t-inherit-mode="extension" owl="1">
         <xpath expr="//Dropdown" position="replace">
             <!-- Same hotkey as in EE -->
-            <AppsMenu
-                hotkey="'a'"
```
Comment (pedrobaeza) [inline web_responsive/static/src/components/apps_menu/apps_menu.xml:10]: Are we losing the hotkey?

**Comment by sbidoul** [comment]
> @sbidoul Last week I'm not able to try anything in runboat. It gives an nginx 503 error. Do you know why?

I've received a few report that the runboat builds are slow to become available. The start eventually, but slowly.
I don't know the cause yet. It started more or less when the mail catcher was deployed, although I don't think it is related.

**Comment by pedrobaeza** [comment]
OK, I see that it's now available. I have waited about 3 minutes with no luck. Let's hope you ditch the problem. Let me know if you want me to test something.

**Review by pedrobaeza — APPROVED** [review]
Tested on runboat and working

**Comment by pedrobaeza** [comment]
Let's fast-track it for fixing the problem:

/ocabot merge patch

**Pull request merged**